### DISCUSSION TITLE
OMI glTF Working Group Meeting 2022-07-21

### DISCUSSION THREAD

**Discussion by robertlong** [omigroup/gltf-extensions#96] [General]
This meeting will be on 7/21/2022 at 10:30 PM (UTC) / 2:30 PM (PST) in the [OMI Discord](https://discord.gg/NJtT9grz5E) within the #omi-weekly-meeting channel. During the meeting, we will be using the #omi-gltf-extensions channel to manage a speaker queue, post links, and for any sidebar discussions.

To be notified of this meeting and others, subscribe to the [OMI Meetings and Events Calendar](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t) or add yourself to the @omi-gltf-subgroup role in the #roles channel of the OMI Discord.

## Agenda:

- [gltf extensions](https://github.com/omigroup/gltf-extensions) Discuss support table
- glTF Transform Updates
- [KHR_audio](https://github.com/KhronosGroup/glTF/pull/2137) Feedback
- [OMI_collider](https://github.com/omigroup/gltf-extensions/pull/63)
  - Jenga demo asset
- OMI_id / [OMI_ref](https://github.com/omigroup/gltf-extensions/pull/85)
- gltf Behavior Graphs Proposal: https://docs.google.com/presentation/d/1zx5cI67cXjlTFzuDId8vc-pU7XDTmKuKi3noIxWCJTY/edit#slide=id.g13e8d9d95f7_20_30
- End of the meeting show and tell period

To propose another item for the agenda, comment below (preferably before the meeting).

**Comment by madjin** [comment]
It'd be worth noting any ideas that come up related to accelerate the testing and adoption of standards that we might wanna consider allocating any budget into (tools or plugins, documentation, perhaps a presentation space, etc)
